ticdc/ticdc-bidirectional-replication.md

```diff
@@ -34,47 +34,113 @@ TiCDC only replicates incremental data changes that occur after a specified time
+
+### Replication scenarios of replicable DDLs
+
+1. Choose a TiDB cluster and execute `ADMIN SET BDR ROLE PRIMARY` to set it as the primary cluster.
+2. On other TiDB clusters, execute `ADMIN SET BDR ROLE SECONDARY` to set them as the secondary clusters.
+3. Execute **replicable DDLs** on the primary cluster. The successfully executed DDLs will be replicated to the secondary clusters by TiCDC.
+
+> **Note:**
+>
+> To prevent misoperations:
+>
+> - If you try to execute **replicable DDLs** on the secondary clusters, you will get an [Error 8263](/error-codes.md).
+> - No matter you try to execute **replicable DDLs** or **non-replicable DDLs** on the secondary clusters, you will get an [Error 8263](/error-codes.md).
+
+### Replication scenarios of non-replicable DDLs
+
+1. Execute `ADMIN UNSET BDR ROLE` on all TiDB clusters to cancel the BDR role.
+2. Stop writing data to the tables that need to execute DDLs in all clusters.
+3. Wait until all writes to the corresponding tables in all clusters are replicated to other clusters, and then manually execute all DDLs on each TiDB cluster.
+4. Wait until the DDLs are completed, and then resume writing data.
+5. Follow the steps in [Replication scenarios of replicable DDLs](#replication-scenarios-of-replicable-ddls) to switch back to the replication scenario of replicable DDLs.
 
 ## Stop bi-directional replication
 
 After the application has stopped writing data, you can insert a special record into each cluster. By checking the two special records, you can make sure that data in two clusters are consistent.
 
-After the check is completed, you can stop the changefeed to stop bi-directional replication.
+After the check is completed, you can stop the changefeed to stop bi-directional replication, and execute `ADMIN UNSET BDR ROLE` on all TiDB clusters.
 
 ## Limitations
 
-- For the limitations of DDL, see [Execute DDL](#execute-ddl).
+- Use BDR role only in the following scenarios:
+
+    - 1 `PRIMARY` cluster and n `SECONDARY` clusters (replication scenarios of replicable DDLs)
+    - n clusters that have no BDR roles (replication scenarios of non-replicable DDLs)
+
+    **Note that do not set the BDR role to other scenarios, for example, setting `PRIMARY`, `SECONDARY`, and no BDR roles at the same time. If you set the BDR role incorrectly, TiDB cannot guarantee data correctness.**
+
+- Usually do not use `AUTO_INCREMENT` or `AUTO_RANDOM` to avoid data conflicts in the replicated tables. If you need to use `AUTO_INCREMENT` or `AUTO_RANDOM`, you can set different `auto_increment_increment` and `auto_increment_offset` for different clusters to ensure that different clusters can be assigned different primary keys. For example, if there are three TiDB servers (A, B, and C) in bi-directional replication, you can set them as follows:
```

Comment:
```suggestion
- Usually do not use `AUTO_INCREMENT` or `AUTO_RANDOM` to avoid data conflicts in the replicated tables. If you need to use `AUTO_INCREMENT` or `AUTO_RANDOM`, you can set different `auto_increment_increment` and `auto_increment_offset` for different clusters to ensure that different clusters can be assigned different primary keys. For example, if there are three TiDB clusters (A, B, and C) in bi-directional replication, you can set them as follows:
```